lib/gooddata/models/schedule.rb

```diff
@@ -526,6 +538,7 @@ def to_update_payload
           'timezone' => timezone,
           'cron' => cron,
           'triggerScheduleId' => trigger_id,
+          'triggerExecutionStatus' => trigger_execution_status ? trigger_execution_status : TRIGGER_EXECUTION_STATUS,
           'params' => GoodData::Helpers.encode_public_params(params),
```

Comment:
Why do we need set default value TRIGGER_EXECUTION_STATUS instead of null ?

Reply:
Because the new Open API ignore other value. The triggerExecutionStatus must have 'OK' or 'OK+WARNING'